docs/src/technologies/compartment.md

```diff
@@ -0,0 +1,13 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Compartmentalization
+Compartmentalization is the technique of separating parts of a system to decrease attack surface and prevent malfunctions from cascading in the system. In Ghaf architecture, there is a separate Virtual Machine (VM) for every vital function of the system.
+
+Current implementation supports Graphic User Interface (GUI) VM, Networking VM and a couple of Application VMs, such as Chromium web-browser, GALA-app and Zathura pdf reader.
```

Comment:
There was discussion to leave GALA-app out as a Ghaf reference app and move it to a ghaf-based template project.
There's good reason for it from Ghaf-perspective - non-source delivery, requires back-end infra with access rights to  Cuttlefish virtual devices. 

At this point, I would not mention it here.

Reply:
Removed the mention about GALA